tests/test_validation.py

```
@@ -11,81 +11,80 @@
 load_dotenv()
 
 
-def create_md5_with_pii(md5: str, emails: list[str], phones: list[str]) -> MD5WithPII:
+def create_md5_with_pii(md5: str, emails: list[str], phones: list[str], sentences: list[str] = ["test sentence"]) -> MD5WithPII:
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)